web/reNgine/tasks.py

```diff
@@ -1666,37 +1666,38 @@ def dir_file_fuzz(self, ctx={}, description=None):
 				logger.error(f'FUZZ not found for "{url}"')
 				continue
 			endpoint, created = save_endpoint(url, crawl=False, ctx=ctx)
-			# endpoint.is_default = False
-			endpoint.http_status = status
-			endpoint.content_length = length
-			endpoint.response_time = duration / 1000000000
-			endpoint.save()
-			if created:
-				urls.append(endpoint.http_url)
-			endpoint.status = status
-			endpoint.content_type = content_type
-			endpoint.content_length = length
-			dfile, created = DirectoryFile.objects.get_or_create(
-				name=name,
-				length=length,
-				words=words,
-				lines=lines,
-				content_type=content_type,
-				url=url)
-			dfile.http_status = status
-			dfile.save()
-			# if created:
-			# 	logger.warning(f'Found new directory or file {url}')
-			dirscan.directory_files.add(dfile)
-			dirscan.save()
-
-			if self.subscan:
-				dirscan.dir_subscan_ids.add(self.subscan)
-
-			subdomain_name = get_subdomain_from_url(endpoint.http_url)
-			subdomain = Subdomain.objects.get(name=subdomain_name, scan_history=self.scan)
-			subdomain.directories.add(dirscan)
-			subdomain.save()
+			if endpoint:
+				# endpoint.is_default = False
```

Comment:
Is this comment still needed?

Reply:
Don't know, I could delete it

Reply:
Not needed, feel free to remove

Reply:
@psyray Remove it please so I can merge it.